recipes/octomap/all/conanfile.py

```diff
@@ -82,6 +85,9 @@ def _patch_sources(self):
         # No -Werror
         if tools.Version(self.version) >= "1.9.6":
             tools.replace_in_file(compiler_settings, "-Werror", "")
+        # we want a clean rpath in installed shared libs
+        tools.replace_in_file(compiler_settings, "set(CMAKE_INSTALL_RPATH \"${CMAKE_INSTALL_PREFIX}/lib\")", "")
+        tools.replace_in_file(compiler_settings, "set(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)", "")
```

Comment:
in https://github.com/conan-io/hooks/issues/376, you don't mention anything about `CMAKE_INSTALL_RPATH`/`CMAKE_INSTALL_RPATH_UE_LINK_PATH`. I think it should be explained why they have to be modified.

Reply:
Yes. Actually these 2 ones have to be left to their default value in CMake, otherwise it leads to hardcoded paths in RPATH of binaries in install tree.